tasks/monitoring.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Copy monit monitor
+  template:
+    src: "templates/etc/monit/conf.d/smpp-host-connection.j2"
+    dest: "/etc/monit/conf.d/smpp-host-connection"
+    mode: 0644
+  notify:
+    -  Reload Monit
+
+- name: Make sure monit exec-scripts directory exists
+  file:
+    path: "/etc/monit/exec-sripts/"
```

Comment:
@morrismukiri there seems to be a typo in the path. Should probably be `/etc/monit/exec-scripts` right?